Workade/Models/RegionModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RegionModel.swift
+//  Workade
+//
+//  Created by 김예훈 on 2022/11/23.
+//
+
+import Foundation
+
+enum RegionModel: String, Hashable, CaseIterable {
+    case gyeonggido
+    case gangwondo
+    case chungcheongbukdo
+    case chungcheongnamdo
+    case jeollabukdo
+    case jeollanamdo
+    case gyeongsangbukdo
+    case gyeongsangnamdo
+    case jeju
+    
+    var name: String {
+        switch self {
+        case .gyeonggido:
+            return "경기도"
+        case .gangwondo:
+            return "강원도"
+        case .chungcheongbukdo:
+            return "충청북도"
+        case .chungcheongnamdo:
+            return "충청남도"
+        case .jeollabukdo:
+            return "전라북도"
+        case .jeollanamdo:
+            return "전라남도"
+        case .gyeongsangbukdo:
+            return "경상북도"
+        case .gyeongsangnamdo:
+            return "경상남도"
+        case .jeju:
+            return "제주도"
+        }
+    }
+    
+    var imageName: String {
+        let backgroundString = "Background"
+        return self.rawValue + backgroundString
+    }
```

Comment:
```suggestion
    var imageName: String {
        return self.rawValue + "Background"
    }
```
[P3] 추가적인 변수명을 둔 이유가 궁금합니다.

Reply:
처음 구현할 당시 해당 모델의 다른 계산 프로퍼티 처럼 switch self { ...} 으로 케이스별로 리턴해주는 방식이어서 여러군데에서 같은 String값을 써 하나의 변수로 빼줬는데 self.rawValue로 한 문장으로 선언가능해져서 따로 빼둘 필요가 없어졌네요. 좋은 지적 감사합니다!